### PR TITLE
feat(AWS EventBridge): Support disabling a rule

### PR DESCRIPTION
**Pull request by jakejscott**
ISSUE: https://github.com/serverless/serverless/issues/9864

### CODE REVIEW

File: lib/plugins/aws/package/compile/events/eventBridge/index.js [hunk 1]
```diff
@@ -235,6 +244,7 @@ class AwsCompileEventBridgeEvents {
         FunctionName,
         EventBridgeConfig: {
           RuleName,
+          State,
```
Comment (pgrzesik) [inline lib/plugins/aws/package/compile/events/eventBridge/index.js:247]: Did you verify that it works for custom-resource-based implementation? I believe it's not going to be honoured at all

File: lib/plugins/aws/package/compile/events/eventBridge/index.js [hunk 2]
```diff
@@ -57,6 +57,7 @@ class AwsCompileEventBridgeEvents {
           ],
         },
         schedule: { pattern: '^(?:cron|rate)\\(.+\\)$' },
+        enabled: { type: 'boolean' },
```
Comment (pgrzesik) [inline lib/plugins/aws/package/compile/events/eventBridge/index.js:60]: It would be great to document this property in docs for `eventBridge` event

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Hello @jakejscott, thanks a lot for proposing this PR. It looks like a good start, but I'm afraid it might not fully work as expected - please see my comments. 

Additionally, it would be great to confirm with tests that the proposed change works.

File: lib/plugins/aws/package/compile/events/eventBridge/index.js (hunk 1 above)
Comment (jakejscott) [reply lib/plugins/aws/package/compile/events/eventBridge/index.js:247]: Cool will take a look at updating that too, thanks :)

File: lib/plugins/aws/package/compile/events/eventBridge/index.js (hunk 2 above)
Comment (jakejscott) [reply lib/plugins/aws/package/compile/events/eventBridge/index.js:60]: Done

File: lib/plugins/aws/package/compile/events/eventBridge/index.js (hunk 1 above)
Comment (jakejscott) [reply lib/plugins/aws/package/compile/events/eventBridge/index.js:247]: @pgrzesik Have updated the custom-resource implementation, but haven't run any tests.

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Thanks for the updates @jakejscott :bow: It looks good, but we still should cover the changes with proper testing by extending test suite here: https://github.com/serverless/serverless/blob/master/test/unit/lib/plugins/aws/package/compile/events/eventBridge/index.test.js

All new tests should be based on `runServerless` utility. Please let me know if you have any questions

**Comment by jakejscott** [comment]
Hi @pgrzesik I've added some unit tests to cover using custom resources or cloudformation.

**Review by pgrzesik — APPROVED** [review]
Thanks @jakejscott, it looks great :raised_hands:

**Comment by jakejscott** [comment]
@pgrzesik We'd love to get our hands on this feature, wondering when the next release is scheduled?

**Comment by pgrzesik** [comment]
Hey @jakejscott - I plan to prepare a new release early morning (CEST) tomorrow.

**Pull request merged**